src/nightwatch/responsive-helpers.js

```diff
@@ -0,0 +1,39 @@
+const screenWidth = (browser) => {
+  const breakpoint = browser.currentTest.name.match(/\[@(.*?)\]/)[1];
```

Comment:
> browser.whenBreakpoint('small', () => {
  // todo assert things when screensize is small
});

At this point with our tests, I do not think a custom command like this would add any value given responsive component has different assertions depending on screen size and the current if-else is pretty simple and easy to read.